### PR TITLE
Add Media and Likes controlers and their tests

### PR DESCRIPTION
**Pull request by yummta**

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def index
+    if params.has_key? :medium_id
+      render json: Like.where(medium_id: params["medium_id"])
```
Comment (liannivin) [inline app/controllers/likes_controller.rb]: Excellen!

File: app/controllers/media_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  def index
+    if params.has_key? :user_id
+      # /users/:id/media
+      render json: Medium.where(user_id: params["user_id"])
+    else
+      # /media
+      render json: Medium.all
+    end
+  end
+end
```
Comment (liannivin) [inline app/controllers/media_controller.rb]: Good work!

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,21 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resources :users
+  resources :users do
+    resources :media, only: [:index] do
+      resources :likes
+    end
```
Comment (liannivin) [inline config/routes.rb]: Could you explain how this code works?

Comment (yummta) [reply config/routes.rb]: Yes, I could. Here I'm nesting resources to this  /users/:id/media can be accessible

**Pull request merged**